### PR TITLE
Update first_steps_1.rst

### PR DESCRIPTION
**Pull request by vicadesoba**
This PR suggestions the addition of some content that will hopefully help new users seamlessly get started with Bokeh. The updates mainly inform the user that Bokeh doesn't have a GUI and prompts them to use simple Python scripts in a Python environment to create outputs they can then view in a web browser.

### CODE REVIEW

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (tcmetzger) [inline docs/bokeh/source/docs/first_steps/first_steps_1.rst]: Optimally, lines should not be longer than 88 characters. I'd add a line break between "the" and "purpose"!

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (tcmetzger) [inline docs/bokeh/source/docs/first_steps/first_steps_1.rst]: In RST, you can use double backticks (``) to format text as code, that's what I'd use for the example file name:

```suggestion
1. Create a new Python file on your machine (e.g. ``simple_line_chart.py``) and open it in a code editor of your choice (such as |Sublime Text|, 
```

Comment (tcmetzger) [inline docs/bokeh/source/docs/first_steps/first_steps_1.rst]: Links in RST are a bit tricky (see https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets).

In this case, I'd use \`Sublime Text`_ in the text and then add the corresponding URL at the bottom of this RST file, i.e. 

```
.. _Sublime Text: https://www.sublimetext.com/
```

Same thing with VS Code: \`Visual Studio Code`_ and then the URL at the bottom of this file:

```
.. _Visual Studio Code: https://code.visualstudio.com/
```

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (bryevdv) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: I'm a little worried that "this guide" might be construed as the entire user guide or docs. I might suggest something a little simpler:

> All the code in "First Steps" sections can be be run as standard Python scripts. 
> When run, these scripts will create HTML outputs that are visible in a web browser.

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (bryevdv) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: Also the references need to be like: 
```
`Sublime Text`_, `Visual Studio Code`_, etc.
```
The syntax with the pipes, e.g. `|bokeh.plotting|` is actually a replacement substitution, not a reference (Sphinx has lots of obscure syntax)

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (bryevdv) [inline docs/bokeh/source/docs/first_steps/first_steps_1.rst]: I think we want `none` here (or possibly `sh` for shell) since this is not, itself, python code
```suggestion
    .. code-block:: none
```

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (bryevdv) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: One last suggestion, I am not sure an entire section heading is needed for this, perhaps it could just be a 
```
.. note::
```
at the end of the previous section?

This also seems advised because with the heading, the previous reference `.. _first_steps_1_line_chart:` is now pointing at the wrong section heading.

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (tcmetzger) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: Yes, this should be `.. code-block:: sh`!

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (vicade-mp) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst:63]: Thanks for all the feedback!

**Comment by vicadesoba** [comment]
Got it, thanks for the feedback!

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (vicadesoba) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: @bryevdv Did you mean at the end of the Installation details section?

Comment (tcmetzger) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: I think pretty much where it is now, just above the "Creating a simple line chart" section. You can find documentation about admonitions and specifically the note admonition here: https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (bryevdv) [inline docs/bokeh/source/docs/first_steps/first_steps_1.rst]:
```suggestion
    Bokeh is a Python library that does not have a graphical user interface (GUI).
```

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (bryevdv) [inline docs/bokeh/source/docs/first_steps/first_steps_1.rst]: trailing space, also line length
```suggestion
1. Create a new Python file on your machine (e.g. ``simple_line_chart.py``) and open
it in a code editor of your choice (such as `Sublime Text`_,`Visual Studio Code`_ , etc.).
```

File: docs/bokeh/source/docs/first_steps/first_steps_1.rst
Comment (bryevdv) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: Actually can we remove this sentence entirely? I really just think this reads more confusing, and that all we need to state is that the code is to be run as a script with `python`

Comment (vicadesoba) [reply docs/bokeh/source/docs/first_steps/first_steps_1.rst]: Got it, sounds good.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Sigh. It looks like there is a new package incompatibility that cropped up today between our pinned version of `autoclasstoc` and sphinx. 
```
  File "/usr/share/miniconda3/envs/bk-test/lib/python3.9/site-packages/autoclasstoc/utils.py", line 165, in filter_attrs
    from sphinx.util.docstrings import extract_metadata
ImportError: cannot import name 'extract_metadata' from 'sphinx.util.docstrings' (/usr/share/miniconda3/envs/bk-test/lib/python3.9/site-packages/sphinx/util/docstrings.py)
```
@vicade-mp this has nothing to do with your PR, but unfortunately we will need to figure out a temporary fix to get the docs building again first, before merging.

**Comment by vicadesoba** [comment]
No worries at all!

**Comment by bryevdv** [comment]
Actually it looks like the main branch CI is also already broken due to this, so there is not any reason to hold off on merging. I can hopefully look at the CI issue later today. Thanks @vicade-mp !

**Review by bryevdv — APPROVED** [review]

**Comment by vicadesoba** [comment]
Thanks @bryevdv!

**Pull request merged**
